### DISCUSSION TITLE
Limit size of log

### DISCUSSION THREAD

**Discussion by briangladu** [mediar-ai/screenpipe#364] [General]
Hi!

Thanks for creating this! Super cool. 

Is there a way to limit the size of the log? Like if I only want Screenpipe to use 200GB of storage? And, if it is limited in that way, what impact will that have? Are logs transferred into a long term memory of sorts? Or does limiting its storage limit the amount of data it can look back at?

**Comment by louis030195** [comment]
@briangladu by log do you mean logs or the mp4 and database data?

atm no

if you mean data, what could be a good experience for you?

**Comment by louis030195** [comment]
hey @briangladu

closing this as part of our cleanup - screenpipe v2.2.0 is now out with major improvements!

check it out: https://x.com/louis030195/status/2014403576475615312

if this is still relevant or you have questions, feel free to reopen this discussion. thanks for being part of the community!
